src/lib/hash-links/use-active-section.ts

```diff
@@ -15,17 +14,22 @@ import getFullNavHeaderHeight from 'lib/get-full-nav-header-height'
  * https://github.com/hashicorp/react-components/pull/325
  */
 export function useActiveSection(
-	headings: TableOfContentsHeading[],
-	isEnabled = true
+	slugs: string[],
```

Comment:
🙋🏻 What's the context behind this change?

Reply:
`useActiveSection` was only using the `slugs` from the provided `headings`. The intent here was to decouple `useActiveSection` from the specific `TableOfContentsHeading` component. That way, when we implement the revised `OutlineNav` component, we'll be able to use this same `useActiveSection` hook.

Reply:
Sounds good! Thank you for the context. 😊